src/hooks/withAppsInMaintenance.jsx

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react'
+import { Registry } from 'cozy-client'
+
+const useAppsInMaintenance = client => {
+  const [appsInMaintenance, setAppsInMaintenance] = useState([])
+  const registry = new Registry({
+    client
+  })
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const appsInMaintenance = await registry.fetchAppsInMaintenance()
+      setAppsInMaintenance(appsInMaintenance)
```

Comment:
I think we should try to keep our hook the more generic we can and `setAppsInMaintenance` outside, but I think I can sleep with it for the moment

Reply:
I don't really see how the `setAppsInMaintenance` could work outside of `useEffect` though? For me fetching the status and setting the state here needs to be coupled.